### PR TITLE
DEV: add plugin outlet to user-menu tabs

### PR DESCRIPTION
**Pull request by chapoi**

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs [hunk 1]
```diff
@@ -30,6 +30,12 @@
             />
           {{/each}}
         </div>
+
+        <PluginOutlet
+          @name="user-menu-tabs-after"
```
Comment (jordanvidrine) [inline app/assets/javascripts/discourse/app/components/user-menu/menu.hbs:35]: Most examples I see in core for before/after outlets have the placement identifier at the front of the name. IE.

` @name="after-create-topic-button"`
`@name="above-latest-topic-list-item"`
`@name="below-latest-topic-list-item-bottom-row"`

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs [hunk 2]
```diff
@@ -30,6 +30,12 @@
...
+          @connectorTagName="span"
+          @outletArgs={{hash plugin=@plugin}}
```
Comment (jordanvidrine) [inline app/assets/javascripts/discourse/app/components/user-menu/menu.hbs:37]: This menu does not have access to the `plugin` here. This should be removed.

A possible helpful argument to add here would probably be user. You can do this like:

`@outletArgs={{hash user=this.currentUser}}`

**Review by jordanvidrine — CHANGES_REQUESTED** [review]
See comments : )

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs (hunk 1 above)
Comment (megothss) [reply app/assets/javascripts/discourse/app/components/user-menu/menu.hbs:35]: > So the name should probably be after-user-menu-tabs

I'd suggest calling this plugin outlet `user-menu-tabs-list__after`.

`user-menu-tabs-list` identifies the placement and using the suffix`__after` ensures it's compatible with a wrapper plugin outlet, should we choose to insert one here in the future.

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs [hunk 3]
```diff
@@ -30,6 +30,12 @@
             />
           {{/each}}
         </div>
+
+        <PluginOutlet
+          @name="user-menu-tabs-after"
+          @connectorTagName="span"
```
Comment (megothss) [inline app/assets/javascripts/discourse/app/components/user-menu/menu.hbs:36]: This is a glimmer component, `connectorTagName` is not really needed here.

**Review by megothss — DISMISSED** [review]

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs (hunk 1 above)
Comment (chapoi) [reply app/assets/javascripts/discourse/app/components/user-menu/menu.hbs]: done, thanks

File: app/assets/javascripts/discourse/app/components/user-menu/menu.hbs (hunk 2 above)
Comment (chapoi) [reply app/assets/javascripts/discourse/app/components/user-menu/menu.hbs:37]: great, will do ty

**Review by davidtaylorhq — APPROVED** [review]

**Pull request merged**